### ISSUE TITLE
Load array of lr_schedulers with priority

### ISSUE THREAD

**Issue by xserban** [NullConvergence/torch-template#19]
In the current version we can use two learning rate schedulers: (1) from pytorch and (2) a custom one. It would be nice to have them in an array of schedulers in the config file, with priorities telling the order. For example, we might want to first set the learning rate at different epochs and afterwards slowly decay it. Then we can pass in the config file 2 schedulers in an array.
